app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationsViewModel.kt

```diff
@@ -153,6 +153,9 @@ class NotificationsViewModel @Inject constructor(
         return when ((notificationViewData as? NotificationViewData.Concrete)?.type) {
             Notification.Type.MENTION, Notification.Type.POLL -> {
                 notificationViewData.statusViewData?.let { statusViewData ->
+                    if (statusViewData.status.account.id == account.accountId) {
```

Comment:
This doesn't need to use the actionable status too?

Reply:
No, because there are no boosts on the notifications page